src/controllers/midi/hss1394controller.cpp

```diff
@@ -168,18 +169,20 @@ int Hss1394Controller::close() {
     return 0;
 }
 
-void Hss1394Controller::sendWord(unsigned int word) {
+void Hss1394Controller::sendShortMsg(unsigned char status, unsigned char byte1,
+                                 unsigned char byte2) {
+    unsigned char channel = MidiUtils::channelFromStatus(status);
+    unsigned char opCode = MidiUtils::opCodeFromStatus(status);
+    unsigned int word = (((unsigned int)byte2) << 16) |
+                         (((unsigned int)byte1) << 8) | status;
```

Comment:
I'd like to get rid of this `word` intermediate, but I don't have any HSS1394 devices nor a Firewire port to test a change.